<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>641</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>状态显示区域</string>
    </property>
    <widget class="QTextBrowser" name="statusBrowser">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>21</y>
       <width>651</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="startSetting">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>启动设置</string>
    </property>
    <widget class="QPushButton" name="detectButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>选择模型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseModelCombo">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择斑马线种类</string>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseZebraCombo">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="videoOptions">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>441</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>视频选项</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择视频类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="chooseVideoType">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="imageOptions">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>441</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>图片选项</string>
     </property>
     <widget class="QCheckBox" name="singleImageCheckbox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>单张图片</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="imageDirCheckbox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>多张图片的目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseImageBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>选择图片</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imagePathLineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件或者目录</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>当前时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="resultArea">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>结果显示区域</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>图片视频显示区域</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>641</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>1121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模式选择</string>
    </property>
    <widget class="QMenu" name="menu_choose_mode">
     <property name="title">
      <string>choose_mode</string>
     </property>
     <addaction name="action_image_mode"/>
     <addaction name="action_video_mode"/>
    </widget>
    <addaction name="menu_choose_mode"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>参数设置</string>
    </property>
    <addaction name="actioncommon_setting"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>快速部署</string>
    </property>
    <addaction name="actionauto_prepare"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondensity_setting">
   <property name="text">
    <string>density setting</string>
   </property>
  </action>
  <action name="actionglobal_setting">
   <property name="text">
    <string>global setting</string>
   </property>
  </action>
  <action name="actionweight_setting">
   <property name="text">
    <string>weight setting</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="action_image_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>image_mode</string>
   </property>
  </action>
  <action name="action_video_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>video_mode</string>
   </property>
  </action>
  <action name="actioncommon_setting">
   <property name="text">
    <string>common setting</string>
   </property>
  </action>
  <action name="actionuse_camera_as_input">
   <property name="text">
    <string>use camera as input</string>
   </property>
  </action>
  <action name="actionuse_video_file_as_input">
   <property name="text">
    <string>use video file as input</string>
   </property>
  </action>
  <action name="actionauto_prepare">
   <property name="text">
    <string>auto prepare</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
